README.md

```diff
@@ -34,16 +36,11 @@ let package = Package(
 }
 ```
 
-### XCode
+## Quickstart
 
-Go to Project Settings -> Swift Packages.
-
-Add a new package and enter: `https://github.com/livekit/client-sdk-swift`
-
-## iOS Usage
-
-LiveKit provides an UIKit based `VideoView` class that renders video tracks. Subscribed audio tracks are automatically played.
+### Something Simple with SwiftUI
```

Comment:
is this a placeholder title?

Reply:
It can be placeholder, or it can be the actual title.